front-end/src/routes/startover/containers/startover.js

```diff
@@ -0,0 +1,13 @@
+import React,{Component} from 'react';
+
+
+export default class StartOver extends Component{
+    render(){
+        return (
+              <div>
+              startovers
```

Comment:
We should create tickets for other dev's to know that these components haven't been made yet, just so they are aware that those components still need to be created

Reply:
sounds good!